File: UserService/Model/Expert.cs
using MongoDB.Bson;
using MongoDB.Bson.Serialization.Attributes;

namespace UserService.Model
{
    public class Expert 
    {
        [BsonId]
        [BsonRepresentation(BsonType.ObjectId)]

        public string? ExpertID { get; set; }

        public string Username { get; set; }
        public string Password { get; set; }
        public string Email { get; set; }
        public string Firstname { get; set; }
        public string Lastname { get; set; }
        public DateTime DateOfBirth { get; set; }
        public string Skills { get; set; }
        public double Rating { get; set; }
        public decimal Budget { get; set; }
        public DateTime? RegistrationDate { get; set; } = DateTime.Now;

    }
}

File: AngularApp/src/models/expert.ts
export class Expert {

   expertId? : string
   username: string
   password: string
   email: string
   firstName: string
   lastName: string
   dateOfBirth: Date
   skills: string
   rating: number
   budget: number

  constructor(){
   this.expertId = ""
   this.username= ""
   this.password= ""
   this.email= ""
   this.firstName= ""
   this.lastName= ""
   this.dateOfBirth= new Date()
   this.skills= ""
   this.rating= 5
   this.budget= 500
  }
    
}